src/schema/rules/tabular_data/modality_agnostic.yaml

```
@@ -0,0 +1,64 @@
+---
+Participants:
+  selectors:
+    - path == "participants.tsv"
+  initial_columns:
+    - participant_id
+  columns:
+    participant_id:
+      level: required
+      description_addendum: |
+        There MUST be exactly one row for each participant.
+    species: recommended
+    age: recommended
+    sex: recommended
+    handedness: recommended
+    strain: recommended
+    strain_rrid: recommended
+  additional_columns: allowed
+
+Samples:
+  selectors:
+    - path == "samples.tsv"
+  columns:
+    sample_id:
+      level: required
+      description_addendum: |
+        The combination of `sample_id` and `participant_id` MUST be unique.
+    participant_id:
+      level: required
+      description_addendum: |
+        The combination of `sample_id` and `participant_id` MUST be unique.
+    sample_type: required
+    pathology: recommended
+    derived_from: recommended
+  additional_columns: allowed
+
+Scans:
+  selectors:
+    - suffix == "scans"
+    - extension == ".tsv"
+  initial_columns:
+    - filename
+  columns:
+    filename:
+      level: required
+      description_addendum: |
+        There MUST be exactly one row for each file.
+    acq_time__scans: optional
+  additional_columns: allowed_if_defined
```

My reading is that they are permitted in the usual way. In general TSV files RECOMMEND adding definitions for new columns, so this is not a departure:

> All such included additional fields SHOULD be documented in an accompanying _scans.json file that describes these fields in detail (see Tabular files).

```suggestion
  additional_columns: allowed
```